ReportingServicesTools/Functions/CatalogItems/Get-RsSubscription.ps1

```diff
@@ -26,6 +26,11 @@ function Get-RsSubscription
             Use "New-RsWebServiceProxy" to generate a proxy object for reuse.
             Useful when repeatedly having to connect to multiple different Report Server.
         
+        .PARAMETER APIVersion
```

Comment:
Please rename this to ApiVersion.

Reply:
Could also please move this after .PARAMETER ReportServerUri?

Reply:
Done, will commit the change shortly.